src/pages/blog.js

```diff
@@ -1,14 +1,23 @@
 import React from 'react';
 import Layout from '../components/Layout';
+import { Link, graphql } from "gatsby"
 
-const Blog = () => {
-  const BlogPost = () => {
-    return (
-      <div>
+const Blog = ({ data }) => {
+  const posts = data.allMarkdownRemark.nodes
+  console.log(posts)
 
-      </div>
+  if (posts.length === 0) {
+    return (
+      <Layout>
+        <p>
+          No blog posts found. Add markdown posts to "content/blog" (or the
```

Comment:
This message doesnt make sense as a user facing blog. Why would they add stuff. Just say something like "Sorry there are no blog posts to be found"